docs/requirements.md

```diff
@@ -1,16 +1,59 @@
-# PerfCheck CI Requirements
+# Perf Check CI Requirements
 
 Perf Check CI is a web application to manage jobs that measure and compare performance on another web application.
 
 ## Creating a job
 
-* As a user I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I want to compare relative performance of my branch against master to make sure I'm not shipping code that will slow things down.
+* As a user, I want to have a quick idea how performance on my branch looks, without caring about comparisons against master.
+* As a user, I want to create a job to compare the performance of two branches across multiple request paths.
```

Comment:
What is the motivation to compare branches?

Reply:
I’ll rephrase this to make it clear it’s specifying non-master/arbitrary  reference branches. 

Its most relevant when having the reference branch be a feature branch and the test branch being a branch with an alternative implementation, for example one that includes perf fixes. It’s less important than master/production branch.